packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -357,6 +357,12 @@ export class StripeWebhookHandler extends StripeHandler {
           invoice,
           error.longMessage
         );
+
+        this.log.error('handleCreditNoteEvent', {
+          invoiceId: invoice.id,
+          message: 'Paypal refund refused.',
+        });
+        reportSentryError(error, request);
```

Comment:
It'd be nice to include the invoice ID in Sentry as well. I don't see a way to do this with `reportSentryError` other than to add an `output.payload` object on the `RefusedError` which is what we do for `AppError` more generally.

There's also [`Sentry.withScope`](https://github.com/mozilla/fxa/blob/d737e414ce0f0657adcb767fbdf5718fa48973bf/packages/fxa-auth-server/lib/devices.js#L198-L202).

Reply:
added